### PR TITLE
designed user list and its responsive too

### PR DESCRIPTION
**Pull request by ChitvanRamani22**
## Related Issuse

- Info about Issue or bug

Closes: #443 

Screenshot (517)
@codewithvk please review this pr.

### CODE REVIEW

File: client/src/components/Admin_userlist/admin_userlist.js [hunk 1]
```diff
@@ -95,50 +81,43 @@ export default function StickyHeadTable() {
     const user = useSelector((state) => state.user);
     const dispatch = useDispatch();
 
-        if (auth.authenticate) {
-        return <Redirect to={"/"} />;
-    }
 
-    if (user.loading) {
-        return <Progress mb='4' w='25%'><Progress.Bar striped animated min='0'max='100' mx='auto' now='50'>Loading....</Progress.Bar></Progress>;
```
Comment (purnima143) [inline client/src/components/Admin_userlist/admin_userlist.js:103]: Why did you remove this line?

Comment (ChitvanRamani22) [reply client/src/components/Admin_userlist/admin_userlist.js:103]: Oh sorry that is unintentional.
Will correct it.

**Comment by ChitvanRamani22** [comment]
@codewithvk @purnima143 i have corrected it.
